src/ServiceControl/Notifications/Email/EmailSender.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ServiceControl.Notifications.Email
+{
+    using System;
+    using System.Net;
+    using System.Net.Mail;
+    using System.Threading.Tasks;
+    using NServiceBus.Logging;
+
+    class EmailSender
+    {
+        public static async Task Send(EmailNotifications settings, string subject, string body, string emailDropFolder = null)
+        {
+            try
+            {
+                using (var client = CreateSmtpClient(settings, emailDropFolder))
+                {
+                    using (var mailMessage = new MailMessage(settings.From, settings.To, subject, body))
+                    {
+                        await client.SendMailAsync(mailMessage).ConfigureAwait(false);
+                    }
+                }
+            }
+            catch (Exception e)
+            {
+                log.Warn("Failure sending email.", e);
+                throw;
+            }
+        }
+
+        static SmtpClient CreateSmtpClient(EmailNotifications settings, string emailDropFolder)
+        {
+            if (emailDropFolder != null)
+            {
+                return new SmtpClient
+                {
+                    PickupDirectoryLocation = emailDropFolder,
+                    DeliveryMethod = SmtpDeliveryMethod.SpecifiedPickupDirectory
+                };
+            }
+
+            var smtpClient = new SmtpClient(settings.SmtpServer, settings.SmtpPort ?? 25)
```

Comment:
Why not have a client that is created when needed, reused until the settings change and disposed when they do or SC is shutdown?

>The connection established by the current instance of the SmtpClient class to the SMTP server may be re-used if an application wishes to send multiple messages to the same SMTP server. This is particularly useful when authentication or encryption are used establish a connection to the SMTP server. The process of authenticating and establishing a TLS session can be expensive operations. A requirement to re-establish a connection for each message when sending a large quantity of email to the same SMTP server could have a significant impact on performance. There are a number of high-volume email applications that send email status updates, newsletter distributions, or email alerts. Also many email client applications support an off-line mode where users can compose many email messages that are sent later when a connection to the SMTP server is established. It is typical for an email client to send all SMTP messages to a specific SMTP server (provided by the Internet service provider) that then forwards this email to other SMTP servers.

> The SmtpClient class implementation pools SMTP connections so that it can avoid the overhead of re-establishing a connection for every message to the same server. An application may re-use the same SmtpClient object to send many different emails to the same SMTP server and to many different SMTP servers.

https://docs.microsoft.com/en-us/dotnet/api/system.net.mail.smtpclient

Reply:
As discussed, this is meant to be rarely executed. Secondly, based on test it looks that there is some caching going on behind the scenes as 2nd and follow-up connections are considerably quicker than the 1st one.